### PR TITLE
Fix Build to Ensure correct version of pyproj and geopandas - remove basemap

### PR DESCRIPTION
**Pull request by nkorinek**
Pinning geopandas version to be greater than 0.7.0. This ran locally just fine so hopefully it doesn't cause any issues.

### CODE REVIEW

**Comment by nkorinek** [comment]
@lwasser

**Comment by lwasser** [comment]
@nkorinek it ran locally but it looks like it's still building to 0.6.3. is that correct? i may post over in the conda-forge repo for geopandas if that is still happening

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN conda update conda --yes \
     branca=0.3.1 \
```
Comment (lwasser) [inline Dockerfile]: We are pinning branca because of an issue with folium. but geopandas was not installing correctly prior to pinning branca

**Comment by lwasser** [comment]
merging this as the build looks good!!!

**Pull request merged**